---
title: Genomic epidemiology reveals multiple introductions followed by community and nosocomial spread of SARS-CoV-2 in Bavaria, February to May 2020.
authors:
  - Max Münchhoff
  - Alexander Graf
  - Stefan Krebs
  - Caroline Quartucci
  - Sandra Hasmann
  - Johannes Hellmuth
  - Clemens Scherer
  - Andreas Osterman
  - Stephan Boehm
  - Christopher Mandel
  - Andrea Becker-Pennrich
  - Michael Zoller
  - Hans Chrisitan Stubbe
  - Stefan Munker
  - Dieter Munker
  - Katrin Milger-Kneidinger
  - Madeleine Gapp
  - Stephanie Schneider
  - Adrian Ruhle
  - Linda Jocham
  - Kami Pekayvaz
  - Tobias Weinberger
  - Helga Mairhofer
  - Elham Khatamzas
  - Katharina Hofmann
  - Patricia Spaeth
  - Sabine Bender
  - Stefan Kääb
  - Bernhard Zwißler
  - Julia Mayerle
  - Juergen Behr
  - Michael von Bergwelt-Baildon
  - Martin Reincke
  - Beatrice Garbein
  - Christian Ludwig Hinske
  - Helmut Blum
  - Oliver T. Keppler

authorLinks:
  - https://author1.org
  - https://author2.io

dataset: "https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/global?c=region&d=tree,entropy&p=full" # must be accessible to the auspice server running the narrative
date: "April 2021"
abstract: " "


---

<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 1 -->
# [Summary](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/global?c=region&d=tree,entropy&p=full)

<!-- This is the left-side text -->

We apply genomic epidemiology to trace SARS-CoV-2 spread in the Munich Metropolitan Area on an international, national and local level in combination with anamnestic reporting of travel history, interpersonal interactions among patients and staff. 

We investigated genome sequences of 174 SARS-CoV-2-positive respiratory samples collected at the LMU Klinikum, in Munich, between 29 February and 27 May 2020 to characterize cluster outbreaks and to establish likely scenarios and time lines of transmissions.


<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 2 -->
<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
# [Phylogenetic relationship of SARS-CoV-2 from the Munich Metropolitean Area and global strains ](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/global?c=region&d=tree,entropy&p=full)

<!-- This is left-side text -->

The phyolgenetic tree shows the 174 collected SARS-CoV-2 genome sequences in the global context.

<!-- There is NO right-side text on this slide -->

<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 3 -->
<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
# [Phylogenetic relationship of SARS-CoV-2 from the Munich Metropolitean Area and global strains ](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/global?c=region&d=tree,entropy&p=full&f_region=Munich)

The Munich SARS-Cov-2 isolates were distributed troughout the tree suggesting multiple independent introductions of SARS-CoV-2 into the Munich Metropolitean Area during the first weeks of the first pandemic wave.


<!-- There is NO right-side text on this slide -->


<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 4 -->
<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
# [Regional phylogenetic relationship of SARS-CoV-2 genomes ](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/regional?c=country&d=tree,entropy&p=full)

<!-- This is left-side text -->

To explore the source of the early introductions we combined SARS-CoV-2 sequence data with detailed patient history.
Most early COVID-19 cases at the LMU Klinikum had a travel history to either Austria or Northern Italy.

The patient’s travel history revealed multiple introductions from various skiing areas in the Alps with subsequent community transmission.


<!-- There is NO right-side text on this slide -->


<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 5 -->
<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
# [Local phylogenetic relationship of SARS-CoV-2 genomes and transmission dynamics](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/local?c=status&d=tree,entropy&p=full)

<!-- This is left-side text -->
Local phyologenetic tree showing the 174 Munich cases grouped by staff and patient.

Genome sequences of SARS-CoV-2 genomic sequences derived from samples obtained from patients and staff members were closely related and cluster together in the phylogeny. 

However, given the low genetic diversity in this early phase of the pandemic, sequence similarity or even identity cannot be used to identify transmission pairs.

<!-- There is NO right-side text on this slide -->


<!-- ############ SLIDE BREAK ############# -->
<!-- SLIDE 6 -->
<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
# [Local phylogenetic relationship of SARS-CoV-2 genomes and transmission dynamics](https://nextstrain.org/community/axgraf/SARS-CoV-2/Munich/local?c=covid19_infection_type&d=tree,entropy&p=full)

<!-- This is left-side text -->
Local phyologenetic tree showing the 174 Munich cases grouped by infection type.

Combining genomic data with detailed temporo-spatial metadata in a genomic epidemiology framework we were able to estimate most likely modes of transmissions for the majority of cases. 

For staff members presumable hospital-acquired infections were relatively high with 54% of cases compared to 9.6% among patients. 

Our analyses show that genomic data can be used to support analysis of  transmission dynamics, but detailed metadata including interaction data between subjects remains fundamental in particular in outbreaks with low genetic diversity of variants circulating in the population.


<!-- There is NO right-side text on this slide -->

